Source/WebCore/dom/TreeScope.cpp

```diff
@@ -54,6 +55,7 @@ namespace WebCore {
 
 struct SameSizeAsTreeScope {
     void* pointers[10];
+    Vector<void*> vector;
```

Comment:
This doesn't seem to be used yet?

Reply:
Those are not used. SameSizeAsTreeScope is merely supposed to have the same size as TreeScope. Since I added a Vector to TreeScope, I added one here too so static assertions don't hit.